README.md

```diff
@@ -165,6 +165,8 @@ type LiveStreamProps = {
   };
   // Mute/unmute microphone
   isMuted: false;
+  // Enables/disables the zoom gesture handled natively
+  nativeZoomEnabled?: boolean;
```

Comment:
Renames to `enablePinchedZoom` or `enableZoomOnPinche` (action first).
You could also set the property as mandatory with default value to `true`

Reply:
Done